pkg/plugins/golang/v4/scaffolds/internal/templates/makefile.go

```diff
@@ -178,12 +178,12 @@ run: manifests generate fmt vet ## Run a controller from your host.
 # If you wish to build the manager image targeting other platforms you can use the --platform flag.
 # (i.e. docker build --platform linux/arm64). However, you must enable docker buildKit for it.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
-.PHONY: docker-build
-docker-build: ## Build docker image with the manager.
```

Comment:
This change could significantly impact our users, as these targets are likely used in CI pipelines and other automated processes. We need to carefully weigh the pros and cons here.

Reply:
yeah, waiting to see if community wants the change before changing the other places